preliz/internal/optimization.py

```diff
@@ -515,21 +515,25 @@ def negative_pdf(x):
     return result.x
 
 
-def find_discrete_mode(dist):
+def find_discrete_mode(dist, bounds=None):
     """
     Find mode for a discrete distribution from its pmf.
 
     Parameters
     ----------
     dist : Distribution
         PreliZ distribution object
+    bounds : tuple, optional
```

Comment:
Are these bounds used anywhere? When do we want to use them? The same goes for `find_mode`. I am asking because `xvals` already takes into account the support of the distribution. The bounds could be used if we know the mode is in a region smaller than the support of the distribution, but I think we currently don't have any case like that.